apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -0,0 +1,433 @@
+                                "getConnector.error.description",
+                                { description: error.response.data.description }
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.error.message"
+                            )
+                        })
+                    );
+                } else {
+                    // Generic error message
+                    dispatch(
+                        addAlert({
+                            description: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.genericError.description"
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.governanceConnectors.notifications." +
+                                "getConnector.genericError.message"
+                            )
+                        })
+                    );
+                }
+            })
+            .finally(() => {
+                setConnectorCategoryLoading(false);
+            });
+    };
+
+    /**
+     * This function returns loading placeholder.
+     */
+    const renderLoadingPlaceholder = (): ReactElement => {
+        return (
+            <Grid.Row columns={ 1 }>
+                <div>
+                    <div 
+                        className="ui card fluid settings-card"
+                        data-testid={ `${testId}-loading-card` }
+                    >
+                        <div className="content no-padding">
+                            <div className="header-section">
+                                <Placeholder>
+                                    <Placeholder.Header>
+                                        <Placeholder.Line length="medium" />
+                                        <Placeholder.Line length="full" />
+                                    </Placeholder.Header>
+                                </Placeholder>
+                                <Divider hidden />
+                            </div>
+                        </div>
+                        <div className="content extra extra-content">
+                            <div className="action-button">
+                                <Placeholder>
+                                    <Placeholder.Line length="very short" />
+                                </Placeholder>
+                            </div>
+                        </div>
+                    </div>
+                    <Divider hidden/>
+                </div>
+            </Grid.Row>
+        );
+    };
+
+    const handleConnectorCategoryAction = (connectorCategoryId: string) => {
```

Comment:
Add return types. Check other methods as well.
```suggestion
const handleConnectorCategoryAction = (connectorCategoryId: string): void => {
```